package/libs/openssl/Makefile

```diff
@@ -91,7 +126,7 @@ define Package/openssl-util
   TITLE+= (utility)
 endef
 
-define Package/openssl-util/conffiles
+define Package/openssl/conffiles
```

Comment:
Are you sure this change is correct?

Reply:
My intention was  to install the configuration file `/etc/openssl.cnf` with libopenssl instead of openssl-util (the file can be used by any app, not just the `openssl` binary, and specifically if you need to configure an engine, e.g. for hardware crypto support.  You've caught a bug here nonetheless, since I haven't changed the installation itself.  Thanks for pointing it out, I'll correct that.

Reply:
I think this has to match a package name like libopenssl or libopenssl and not the Makefile name.